benchmark/accuracy/write_ground_truth_catalog_csv.jl

```diff
@@ -15,62 +15,29 @@ GALAXY_ONLY_COLUMNS = [
     :angle_deg,
 ]
 
-# The truth file comes from CasJobs. The query that generates it appears
-# below. I selected the RA/Dec range in this query to include all of the
-# (4263, 5, 119) rcf, but not a lot else. In the query, `run` is limited
-# to 106 and 206 because these are the coadd runs. The nested select
-# structure is important for getting this query to complete in time.
-"""
-#declare @BRIGHT bigint set @BRIGHT=dbo.fPhotoFlags('BRIGHT')
-declare @EDGE bigint set @EDGE=dbo.fPhotoFlags('EDGE')
-declare @SATURATED bigint set @SATURATED=dbo.fPhotoFlags('SATURATED')
-declare @NODEBLEND bigint set @NODEBLEND=dbo.fPhotoFlags('NODEBLEND')
-declare @bad_flags bigint set
-@bad_flags=(@SATURATED|@BRIGHT|@EDGE|@NODEBLEND)
-
-select *
-from (
-select
-  objid, rerun, run, camcol, field, flags,
-  ra, dec, probpsf,
-  psfmag_u, psfmag_g, psfmag_r, psfmag_i, psfmag_z,
-  devmag_u, devmag_g, devmag_r, devmag_i, devmag_z,
-  expmag_u, expmag_g, expmag_r, expmag_i, expmag_z,
-  fracdev_r,
-  devab_r, expab_r,
-  devphi_r, expphi_r,
-  devrad_r, exprad_r
-into coadd_field_catalog
-from stripe82.photoobj
-where
-  run in (106, 206) and
-  ra between 0.449 and 0.599 and
-  dec between 0.417 and 0.629) as tmp
-where
-  ((psfmag_i < 22 and probpsf = 1) or (probpsf = 0 and (expmag_i < 22 or devmag_i < 22))) and
-  (flags & @bad_flags) = 0
-"""
```

Comment:
Oh yeah, I need to remove the 2-3 other places where this thing appears in the code. Will do on next PR.